server/embedded/src/main/resources/log4j2.xml

```diff
@@ -274,11 +274,11 @@ account managers so that we can coordinate edits with other customized copies of
             <AppenderRef ref="INDEXED_DOCUMENTS"/>
         </Logger>
 
-        <Logger name="org.labkey.api.util.DebugInfoDumper" level="debug">
+        <Logger name="org.labkey.api.util.DebugInfoDumper" additivity="false" level="debug">
             <AppenderRef ref="THREAD_DUMP"/>
         </Logger>
 
-        <Logger name="org.labkey.core.admin.AdminController.ContentSecurityPolicyReportAction"  additivity="false" level="warn">
+        <Logger name="org.labkey.core.admin.AdminController.ContentSecurityPolicyReportAction" additivity="false" level="warn">
```

Comment:
Removing `additivity` and the CONSOLE AppenderRef should give us both console and CSP_REPORT logging.